lib/runtime-core/src/import.rs

```diff
@@ -41,7 +41,13 @@ impl IsExport for Export {
 /// }
 /// ```
 pub struct ImportObject {
-    map: Rc<RefCell<HashMap<String, Box<dyn LikeNamespace>>>>,
+    map: Rc<RefCell<HashMap<(String, String), Box<dyn IsExport>>>>,
```

Comment:
Main issue with this is that we need to keep imported instances alive. I'm not sure the best way to do this, however.

Reply:
Hmm, I'm not familiar enough with the code base yet to understand how this change affects the lifetimes of the Exports.  It seems like it should be the same, unless you mean that we were dropping entries by namespace before 🤔

Reply:
Yeah, the `Box<dyn LikeNamespace>` could contain an `Instance`. When getting a function pointer from an `ImportObject`, if the Instance isn't maintained in the `ImportObject`, the function pointer will dangle.

Reply:
Hmm, it's possible to use two maps and check both to emulate a global name lookup.

Though, I'm still a bit unclear on the ownership... `ImportObject`s own `Instance`s and `Instance`s own `ImportObject`s?